WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -14,7 +14,16 @@ struct ItemListView: View {
     @Binding var searchTerm: String
 
     private var analyticsTracker: PointOfSaleItemListAnalyticsTracker {
-        PointOfSaleItemListAnalyticsTracker(itemListType: selectedItemListType)
+        switch selectedItemListType {
```

Comment:
It feels like the changes in this file could benefit from unit testing.

Reply:
Good point. I moved out initialization logic from `ItemListView` and added `PointOfSaleItemListAnalyticsTrackerTests` and `POSItemActionHandlerFactoryTests` that test this logic.